robot_names:
  - Husky1
  - Husky2
  - Husky3
  - Iris

button_robot_map:
  A: Husky1
  B: Husky2
  X: Husky3
  Y: Iris

flashlight_service_suffixes:
  - /downward_flashlight/flashlight_switch
  - /left_flashlight/flashlight_switch
  - /right_flashlight/flashlight_switch

robot_address_map:
  Husky1: 128.0.0.10
  Husky2: 128.0.0.11
  Husky3: 128.0.0.12
  Iris: 128.0.0.13

select_led_service_suffixes:
  - /left_led/led_switch
  - /right_led/led_switch

comm_led_service_suffixes:
  - /center_led/led_switch
